locales/en-US/learn.ftl

```diff
@@ -61,3 +61,77 @@ learn-nomicon-button = Read the ’nomicon
+learn-play-button = Try Rust without installing
+
+learn-install-rustup-header = Rustup: the Rust installer and version management tool
+learn-install-rustup = The primary way that folks install Rust is through a tool called Rustup, which is a Rust installer and version management tool.
+learn-install-rustup-button = Learn more about installation
+
+
+learn-install-cargo-header = Cargo: the Rust build tool and package manager
+learn-install-cargo = <p>When you install Rustup you’ll also get the latest stable version of the Rust build tool and
+                          package manager, also known as Cargo. Cargo does lots of things:</p>
+                        <ul>
+                          <li>build your project with <code>cargo build</code></li>
+                          <li>run your project with <code>cargo run</code></li>
+                          <li>test your project with <code>cargo test</code></li>
+                          <li>build documentation for your project with <code>cargo doc</code></li>
+                          <li>publish a library to <a href="https://crates.io">crates.io</a> with <code>cargo publish</code></li>
+                        </ul>
+                        <p>To test that you have Rust and Cargo installed, you can run this in your terminal of choice:</p>
+                        <p><code>cargo --version</code></p>
+learn-install-cargo-button = Read the cargo book
+
+learn-install-other = Other tools
+learn-install-editors = Rust support is available in many editors:
+learn-install-fmt-clippy = You can install a code formatting tool (Rustfmt) with <code>rustup component add rustfmt</code>, and a linting tool (Clippy) with <code>rustup component add clippy</code>.
+
+
+learn-generating = Generating a new project
+learn-generating-steps = <p>Let’s write a small application with our new Rust development environment. To start, we’ll use Cargo to
+      make a new project for us. In your terminal of choice run:</p>
+    <p><code>cargo new hello-rust</code></p>
+    <p>This will generate a new directory called <code>hello-rust</code> with the following files:</p>
+    { $tree }
+    <p><code>Cargo.toml</code> is the manifest file for Rust. It’s where you keep metadata for your project, as well as
+      dependencies.</p>
+    <p><code>src/main.rs</code> is where we’ll write our application code.</p>
+    <hr/>
+    <p><code>cargo new</code> generates a "Hello, world!" project for us! We can run this program by moving into the new directory that we made and running this in our terminal:</p>
+    <p><code>cargo run</code></p>
+    <p>You should see this in your terminal:</p>
+    { $output }
+
+learn-dependencies = Adding dependencies
+learn-dependencies-steps = <p>Let’s add a dependency to our application. You can find all sorts of libraries on <a href="https://crates.io">crates.io</a>, the package registry for Rust. In Rust, we often refer to
+      packages as “crates.”</p>
```

Comment:
Could this linebreak be removed? Random linebreaks like this look kinda weird in Pontoon especially since the line above is so long.